infrastructure/docker/services/php/frontend/etc/service/.gitignore

```diff
@@ -0,0 +1 @@
+*/supervise
```

Comment:
I don't understand this move too

Reply:
You don't want to commit file created by runsvc as it contains runtime data (pid / state / ...)